### DISCUSSION TITLE
UI Improvements

### DISCUSSION THREAD

**Discussion by NTDLS** [NTDLS/Katzebase#82] [General]
@ingted 
Got a good deal of work to do, but I've been working on the UI. kind of excited! 😄 
Also cleaned up a ton of terrible UI code, but I have a lot of restructuring to do before I release.

Added additional options:
![image](https://github.com/user-attachments/assets/b2c74e12-7467-4312-849a-9ad1c4142a06)

Added code folding:
![image](https://github.com/user-attachments/assets/f871a76f-25ef-4ba1-a01a-efc0109c2fa8)

Adding function explorer, with descriptions (loaded from the server).
![image](https://github.com/user-attachments/assets/42988c18-b289-44c5-81a7-b9cbb4164f2c)

Added results and export grid options:
![image](https://github.com/user-attachments/assets/b76883a0-7460-4fd9-841a-eb85ce9e04ff)

**Comment by ingted** [comment]
ULTRA COOL!!!
